tests/test_addons.py

```diff
@@ -174,7 +174,7 @@ def update_settings(self, settings):
                 addon = LoggedAddon()
                 build_from_crawler_mock.return_value = addon
                 crawler = get_crawler(settings_dict=settings_dict)
-                logger_mock.info.assert_called_once_with(
+                logger_mock.info.assert_called_with(
```

Comment:
This is an issue, this message should not be logged more than once. We need to prevent the logging of this when the `SPIDER_MODULES` setting is loaded.